### PR TITLE
Update about-us.html

### PR DESCRIPTION
**Pull request by mrmshrb**
The pictures for Charmi, Mariam, and Vincent are just placeholders, we just wanted to see how they'd look with pictures.

### CODE REVIEW

**Comment by ramaduraianirudh9867** [comment]
Members should be in alphabetical order - Aparna, Charmi, Kelly, Keshav, Mariam, Vincent

File: about-us.html [hunk 1]
```diff
@@ -63,7 +89,71 @@ <h4><b>Keshav Garg</b></h4>
...
+            <h4><b>Vincent Li</b></h4>
+            <p class="caption"></p>
+          </div>
+        </div>
+      </th>
+    </table>  
+    
   </table>
+     <h1 id="getintouch">
+      Get in Touch
```
Comment (ramaduraianirudh9867) [inline about-us.html:145]: Are you making a built-in "Get in Touch" form where all responses will go directly to the officers? How does this work? @mrmshrb @charmi7

**Comment by ramaduraianirudh9867** [comment]
@Yassin-Kortam2 needs to upload Charmi, Mariam, and Vincent pictures ASAP. Can you both, @mrmshrb @charmi7, just put those potential file names instead of Aparna Square for both of you and Vincent?

File: about-us.html (hunk 1 above)
Comment (Yassin-Kortam2) [reply about-us.html:145]: No, thats contact info card. Read the code.

**Pull request merged**